src/audio/kpb.c

```diff
@@ -834,7 +833,7 @@ static uint64_t kpb_draining_task(void *arg)
 			comp_update_buffer_produce(sink, size_to_copy);
 	}
 
-	time_end =  platform_timer_get(platform_timer);
+	time =  platform_timer_get(platform_timer) - time;
```

Comment:
I expected gcc to complain here, but turns out it warns about useless writes only if there is no read at all, doesn't matter when...

Reply:
@jajanusz can you paste error here. It should be fixed by using ```if (0)``` as my review below.

Reply:
@lgirdwood I meant that I expected warn/error, but there isn't, so gcc disappointed me ;) In many compiles useless writes are warns.